isis/src/base/apps/mappt/main.cpp

```diff
@@ -157,15 +158,24 @@ void IsisMain() {
   Brick b(1, 1, 1, icube->pixelType());
   int intSamp = (int)(proj->WorldX() + 0.5);
   int intLine = (int)(proj->WorldY() + 0.5);
+  
   b.SetBasePosition(intSamp, intLine, 1);
   icube->read(b);
+  
+  QString filterName = "Null";
+  PvlGroup bandBin = icube->label()->findObject("IsisCube").findGroup("BandBin");
+  if (bandBin.hasKeyword("FilterName")) {
+      filterName = bandBin.findKeyword("FilterName")[0];
+  }
```

Comment:
From what I understand the `BandBin` group isn't super homogenized because of physical differences between instruments so this change seems questionable to me.

Reply:
I am also dubious of this change. We could pull the band value from all array keywords in the BandBin group.

Reply:
If you're suggesting is getting DNs for every band, I am not sure if this solves the problem in Laz's original issue of wanting more than a physical band number. He seems fine with only one band output. 

I just don't know enough about how ISIS handles multiple bands to answer whether filter name is the only thing useful here? If I report FilterNumber (e.g. in THEMIS images), is filter number and physical band number always going to be the same therefore a useless thing to report? Are there other ways that bands are reported that might be useful to consider? Laz only cares about filter name so that's all I included but might as well make this more generic if it's easy enough to do and I am already in the code.

Reply:
No, I'm not talking about pulling for every band. The BandBin group works by having array-valued keywords where the values at the same index are for the same band. So, the second value in each keyword is for the second band. Instead of just reporting filter name, you could copy the first value from each array-valued keyword in the BandBin group to the output.

For example here's the BandBin group from a TOG CaSSIS mosaic:

```
  Group = BandBin
    FilterName = (BLU, NIR, PAN, RED)
    Center     = (485, 985, 675, 840) <nm>
    Width      = (165, 220, 250, 100) <nm>
    NaifIkCode = (-143424, -143423, -143421, -143422)
  End_Group
```

You would then put the following in the output:
```
FilterName = BLU
Center     = 485 <nm>
Width      = 165 <nm>
NaifIkCode = -143424
```

Reply:
I considered that but my only concern there is having an output params change based on the cube making outputs a pain to parse, at that point I might as well slap the bandbin group with only the first virtual band at the end of results (if only nested groups were a thing).  

I can make the change and if no one else objects\has a better solution then I don't have an issue either.

Reply:
True, parsing output could be a pain. I think this can just stay as is then.